imap_upstream.adoc

```diff
@@ -16,23 +16,29 @@ IMAP related configuration options are available:
 
 The available balancing algorithms are as follows:
 
-* ClientIpHash
+* AccountIdHash
 
-  Selects an IMAP server server based on a hash of the client IP address.
-  The hashing algorithm used is a simple modulus of the client's ip address as an integer with the number of
-   configured IMAP servers. This will always return the same 'index' for the same client ip address.
-
-   Note: Extra configuration is necessary if a NAT proxy is being utilized in the environment.
+  Selects an IMAP server server based on a hash of the account id of
+  the user making the request.  The hashing algorithm used is a simple
+  modulus of the account id as an integer with the number of
+  configured IMAP servers for the mailbox hosting the account. This
+  will always return the same 'index' for the same account id.
 
 * custom
 
   Selects a registered load balancing algorithm with the following format:
 
       custom:{handler-algorithm} [arg1 arg2 ...]
 
+IMPORTANT: Custom hashing algorithms are required to consistently hash to the same
```

Comment:
I'd suggest that you do this as an admonition block (to make the text formatting a bit cleaner and to avoid pushing text to the right to continue on multiple lines):
```
[IMPORTANT]
_your text here_ (left aligned and line wrapped at col 70)
```

Reply:
ok.  I thought `IMPORTANT:` and `[IMPORTANT]` did the same thing?

Reply:
I'll see what happens :)

Reply:
Oops.  You are correct. My issue was with the indented text, and I expected that this was done to continue the admonition, but it continues until a blank line either way. So ultimately, I'd just prefer the entire attribution be left aligned instead of heavily indented (but that's just my preference - others probably have differing opinions).